tests/Behat/Context/Setup/OrderContext.php

```diff
@@ -27,4 +27,14 @@ public function orderHasBeenPlacedInChannel(OrderInterface $order, ChannelInterf
 
         $this->orderManager->flush();
     }
+
+    /**
+     * @Given /^(this order) has not been placed$/
+     */
+    public function orderHasNotBeenPlaced(OrderInterface $order): void
```

Comment:
```suggestion
    /**
     * @Given /^(this order) has no number assigned$/
     */
    public function orderHasNoNumberAssigned(OrderInterface $order): void
```
Let's be more specific